src/modules/launcher/Wox.Plugin/Common/DefaultBrowserInfo.cs

```diff
@@ -114,6 +114,15 @@ public static void Update()
                         commandPattern = GetIndirectString(commandPattern);
                     }
 
+                    // workaround for firefox installed through Microsoft store
+                    const string FirefoxExecutableName = "firefox.exe";
+                    if (commandPattern.Contains(FirefoxExecutableName) && commandPattern.Contains(@"C:\Program Files\WindowsApps\") && (!commandPattern.StartsWith('\"')))
+                    {
+                        var pathEndIndex = commandPattern.IndexOf(FirefoxExecutableName, StringComparison.Ordinal) + FirefoxExecutableName.Length + 1;
+                        commandPattern = commandPattern.Insert(0, "\"");
+                        commandPattern = commandPattern.Insert(pathEndIndex, "\"");
```

Comment:
Have you tested this? Doesn't the first insert make it so that the following insert adds the `"` in the wrong place, since you've changed it already?

Reply:
Yes, tested, this is why the +1 is there.
But I can invert it and it is cleaner and clearer